debian/control

```diff
@@ -1,9 +1,10 @@
 Source: nicotine
 Section: net
 Priority: optional
-Maintainer: Kip Warner <kip@thevertigo.com>
+Maintainer: Nicotine+ Team <nicotine-team@lists.launchpad.net>
 Standards-Version: 3.9.7
```

Comment:
This should be bumped.

Reply:
I'll do this sometime in March, before end of standard support for Xenial.